lib/drivers/medtronic600/medtronic600Driver.js

```diff
@@ -1767,6 +1739,9 @@ class MM600SeriesDriver extends BCNLDriver {
       .send(this.hidDevice);
     debug(`*** EXPECT CGM HISTORY FROM ${response.dataStart.toDate()} TO ${response.dataEnd.toDate()}`);
     const expectedCgmHistorySize = response.historySize;
+    if (response.dataEnd.toDate() < history.dataEnd) {
```

Comment:
Can you maybe put a comment in here to explain what this does? Does this mean that the delta upload will re-read everything since the last CGM data point? What will happen if you stop using a CGM sensor but continue to use the pump?